ui/app/adapters/mfa-login-enforcement.js

```diff
@@ -0,0 +1,14 @@
+import ApplicationAdapter from './application';
+
+export default class KeymgmtKeyAdapter extends ApplicationAdapter {
+  namespace = 'v1';
+
+  pathForType() {
+    return 'identity/mfa/login-enforcement';
+  }
+
+  async query(store, type, query) {
+    const url = this.buildURL(type.modelName, null, null, 'query', query);
```

Comment:
curious why null, null here. Maybe give example in a comment what this URL looks like. Normally it's not necessary, but this one is hard to read.

Reply:
The second argument to `buildURL` is `id` and the third is `snapshot`. When doing a query both are not available which is why I'm explicitly passing null. I think for this particular adapter, rather than using `buildURL` we could use `urlForQuery` to achieve the same thing and it would read more clearly.